roles/cobbler/templates/snippets/cephlab_packages_rhel

```diff
@@ -1,3 +1,8 @@
 ## {{ ansible_managed }}
+#set os_version = $getVar('os_version','')
```

Comment:
Yeah I'm not sure what to say; we don't usually write docs for APIs we're simply using...